### PR TITLE
get all metadata with searchIdentifier and value

### PR DESCRIPTION
**Pull request by nireeshT**

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java [hunk 1]
```diff
@@ -48,4 +56,12 @@ public String create(MetadataCreateRequest request) throws HygieiaException {
         return entity.getId().toString();
     }
 
+    @Override
+    public DataResponse<Iterable<Metadata>> search(String searchKey, String value) throws HygieiaException {
+         LOGGER.info("MetdataService.search() :  Enter");
```
Comment (benj58xu) [inline src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java]: This log line does not help debugging, as it has no information to the specific instance of search

File: src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java [hunk 2]
```diff
@@ -48,4 +56,12 @@ public String create(MetadataCreateRequest request) throws HygieiaException {
...
+        Iterable<Metadata> mts = customRepositoryQuery.findAllMetaDataBySearchQuery(searchKey,value);
+        return new DataResponse(mts, System.currentTimeMillis());
```
Comment (benj58xu) [inline src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java]: Can mts be null?

Comment (nireeshT) [reply src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java]: @benj58xu mts will not be null, but it can be empty collection.

Comment (nireeshT) [reply src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java]: @benj58xu  logging serach key with value and duration.

**Pull request merged**